packages/server/lib/screenshots.ts

```diff
@@ -316,6 +333,7 @@ const ensureSafePath = function (withoutExt, extension, overwrite, num = 0) {
     }
 
     // path does not exist, attempt to create it to check for an ENAMETOOLONG error
+    // @ts-ignore
```

Comment:
This `@ts-ignore` should be able to be removed by changing the next line to `fs.outputFile` - according to the `fs-extra` version we're using, its methods can either be callback-style or promise style. https://github.com/jprichardson/node-fs-extra/tree/9.1.0?tab=readme-ov-file#sync-vs-async-vs-asyncawait

(this also means we don't need to promisify everything in `server/lib/util/fs.ts`, I believe)

Reply:
[`7d6e3a0` (#30758)](https://github.com/cypress-io/cypress/pull/30758/commits/7d6e3a0b960a487f596937ecb85371a3e39534eb)